devnet/README.md

```diff
@@ -23,6 +23,39 @@ docker-compose down
 
 > **_NOTE:_**  The devnet is not stateful, so subsequent runs will start from the genesis block.
 
+## Running Cross Client Devnet
+
+A cross-client devnet is network of gossamer node(s) and substrate node(s).
+
+Steps to run a two node cross client devnet
+
+- Create `genesis-spec.json`
+```
```

Comment:
nit [markdown linters](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) like an empty line before a code block. Same for other code blocks.

````suggestion

```
````